_posts/2017-09-26-plugin-api-alpha.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Pulp 3 Plugin API Alpha
+author: Michael Hrivnak
+tags:
+    -3.0
+---
+
+You are invited to try the first alpha release of the Pulp 3 Plugin API! Pulp 3
+includes a dedicated python package for the plugin API. This package is
+versioned separately to communicate backward-compatibility, and it enables
+developers to write plugins in as simple and straight-forward a way as
```

Comment:
s/a way/way/

Reply:
I think this is correct. The full form would be "simple and straight-forward of a way as possible", so I'll add the "of" back in for clarity.